backend/src/services/release.ts

```diff
@@ -295,6 +335,142 @@ export async function getReleaseBySemver(user: UserInterface, modelId: string, s
   return release
 }
 
+function getSemverQueryBounds(querySemver: string) {
+  const semverRangeStandardised = semver.validRange(querySemver, { includePrerelease: false })
+
+  if (!semverRangeStandardised) {
+    throw BadReq(`Semver range, '${querySemver}' is invalid.`)
+  }
+
+  const [expressionA, expressionB] = semverRangeStandardised.split(' ')
+
+  let lowerInclusivity, lowerSemver, upperInclusivity, upperSemver
+
+  //If lower semver
+  if (expressionA.includes('>')) {
+    lowerInclusivity = expressionA.includes('>=')
+    lowerSemver = expressionA.replace(/[<>=]/g, '')
```

Comment:
To avoid having multiple variables for the upper and lower semvers and another set of if statements after this one, could you include `.replace(/[<>=]/g, '')` at the start of the function `semverStringToObject(0`? It would also mean you'd only need to have this replace in one place

Reply:
Done. I still believe I need all 3 replaces as I don't know whether expressionA is lower or upper.